docs/docs/rest-api/public/api/v2/types/podContainer.raml

```diff
@@ -97,7 +99,7 @@ types:
           Values here override a "user" value specified in the pod definition.
       healthCheck?:
         type: health.HealthCheck
-        description: All healthchecks to perform on the container
+        description: All health checks to perform on the container
```

Comment:
"The health check to perform on the container, if any." – there can be at most one.

Reply:
that will teach me to fix misspellings :)